tests/microos/cockpit_service.pm

```diff
@@ -49,6 +49,7 @@ sub run {
 
     if (@pkgs) {
         record_info('TEST', 'Installing Cockpit\'s Modules...');
+        assert_script_run('zypper ref');
```

Comment:
I suggest to at least add a comment here that the explicit zypper ref might help to work around some refresh problems. *If* it helps then I would still regard it as a product regression and should be reported as such.